src/components/Comment/Comment.scss

```diff
@@ -0,0 +1,37 @@
+.commectCard {
+	margin-bottom: 15px;
+	
+	.commentHeader {
+		display: flex;
+		flex-direction: row;
+	}
+
+	.imagePlaceholder {
+		width: 50px;
+		height: 50px;
+		border-radius: 50px;
+		background-color: #ddd;
+		display: inline-block;
+	}
+	.authorData {
+		margin-left: 10px;
+
+		.commentDate {
+			font-size: 12px;
+		}
+	}
```

Comment:
@ezrogha  change the px to rem

Reply:
Thanks for the feedback @felixkiryowa,
I has been implemented